exec-tests

```diff
@@ -164,6 +164,7 @@ if [[ "${subtst:-python}" == "python" ]]; then
     # used which are not compatible with SQLAlchemy 2.x. Since we are not
     # going to switch to 2.x any time soon, we use the big hammer approach
     # to avoid the noise.
+    python3 -m pip list
```

Comment:
Are we planning on keeping this?  (The log file is too big as is, so I'd prefer not to add to it if we don't have to....)

Reply:
When something goes wrong, it's important to know. I miss the old list that either tox or pytest used to generate. We could try for a more compact format, but the information is valuable. (One simple modification would be `python3 -m pip list --format freeze|column` which will give a multi-column `<package>=<version>` format.)

Reply:
The principal issue with the log is the overall size (i.e., number of bytes) not the number of lines, so reformatting the output and arranging it into columns might not help.

> Counter guy:  "Would you like your pizza cut into six slices or eight?"
> 
> Customer:  "Oh, you better make it six -- I don't think I can eat eight!"

What I would like to have is for the details to be shunted into auxiliary log files, which we could pull in the event of a problem, and have the main log just say "OK, I'm doing this; now I'm doing this; now that's done, and I'm on to the next thing" -- I really don't want to see the gory details of building RPMs and installing them in containers in all of the permutations which we support.

Reply:
I always wanted to have the container journal in an artifact we could access at need; it's really hard to read inline anyway. And, yeah, same for a lot of other things. But none of that will affect the "total bytes" either, so it seems you're in effect recognizing that part of the issue is the vertical distance to scan: and columnizing the output *would* help with that. (But I experimented a bit and the real problem is that `tox` doesn't appear to propagate my `COLUMNS` even when I add it to `passenv`, and therefore `column` doesn't do anything; and while I can set it to an arbitrary value, that's not really satisfying since it's likely to be either longer or shorter than the actual terminal line interactively.)

But since you approved, I'm going to just merge this so Siddardh can move ahead, and we can figure out what to do with all of our logging going forward...

Reply:
And we really need to figure out that annoying tool meister legacy failure. 😦

Reply:
> none of that will affect the "total bytes" either, so it seems you're in effect recognizing that part of the issue is the vertical distance to scan

The vertical distance is mostly an issue for the human with the mouse and the screen.  The total size impacts Mr. Jenkins' behavior.  Part of that is in his screen presentation (I don't recall the details now, but I seem to recall him complaining about the size), and part of that is the workspace impact.  You're correct that shunting stuff to secondary files won't help with the total disk space, but it would help with the other problems (although, we need to take care in how we do it, so that we can view more than just the latest build...).

> since you approved

I didn't want to block the merge if you had a strong preference to keep it, but, if you had intended to remove it, I wanted to make sure that you had the opportunity.  🙂 

> we really need to figure out that annoying tool meister legacy failure.

Yeah.  I'm hoping that that will happen one way or another when we turn our attention to Arcaflow integration....